src/core_atmosphere/Registry.xml

```diff
@@ -333,6 +333,12 @@
                      units="-"
                      description="Whether to re-compute coupled fields $\theta_m$, $\tilde{\rho}$, $\rho u$, etc. from uncoupled fields when restarting the model; used for cycling DA experiments that analyze uncoupled fields in restart files"
                      possible_values=".true. or .false."/>
+
+                <nml_option name="config_use_2streams_ic" type="logical" default_value="false" in_defaults="false"
+                     units="-"
+                     description="Whether use separate_init_stream or not"
```

Comment:
`description="Whether to use separate init and static streams.  Otherwise a single init stream is used for all variables."`

Reply:
How about change to :
“Whether to use static and input streams.”
I prefer to use “input” instead of “init”, because the stream name is “input”.
Input stream (or you call it init stream) does not include time-invariant fields (the mesh, terrain variables), we should use input stream with static stream.

This new functionality does not work with restart input.  In skamaroc@096b5d3, restart stream does not include the time-invariant variables, but in this PR, I have not touched restart stream. This is why this PR has no impact on restart mode run.

Reply:
I agree with `input`.  Thank you for clarifying.

Reply:
OK, sorry to be very particular about this.  I misunderstood your comment to only include the text, `Whether to use static and input streams`.  I think that is too short and not clear.  For example, the descriptions of `config_do_restart` and `config_do_DAcycling` are much more clear because they speak to what is different about those initialization procedures.  Maybe one sentence about the `static` stream would suffice.  In the PR description, you have `Includes the mesh, some of sfc_input variables(landmask, shdmin, albedo12m, etc) and parameters for gravity wave drag over orography.` That information would be useful here as long as the terminology and variable names used in the description are consistent with the remainder of `Registry.xml`.